test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -190,9 +196,10 @@ public void cannotCreateTheSameEmailAccountTwice() {
         try {
             authService.signUp(user.getStudy(), user.getSignUp(), false);
             authService.signUp(tempStudy, user.getSignUp(), false);
-            fail("Should not get here");
+            
+            verify(authService).requestResetPassword(any(Study.class), any(Email.class));
         } catch (EntityAlreadyExistsException e) {
-            assertEquals("Account already exists.", e.getMessage());
+            fail("Should not throw entity already exists");
```

Comment:
At this point there's no exception or http status code... but I check this in the accompanying integration test SDK PR.